package.json

```diff
@@ -23,5 +24,8 @@
     "express": "^4.14.0",
     "socket.io": "^1.7.2",
     "ejs": "^2.5.5"
+  },
+  "devDependencies": {
+    "nodemon": "^2.0.5"
```

Comment:
Do not add any other dependency. I don't think we are having to close server and restart the app after making any changes, you just need to refresh the tab.